ncm-metaconfig/src/main/metaconfig/conntrackd/pan/schema.pan

```diff
@@ -0,0 +1,117 @@
+declaration template metaconfig/conntrackd/schema;
+
+include 'pan/types';
+
+@documentation {
+    Conntrackd config
+}
+
+@documentation {
```

Comment:
So I've just noticed that the space here is being flagged by Panlint and is not being understood by the syntax highlighting. The Pan documentation shows annotations without any white-space, but the compiler accepts it. I can't decide whether I should update Panlint and the syntax highlighting or whether I should ask you to remove the space.